src/main/java/com/stripe/model/checkout/Session.java

```diff
@@ -77,6 +78,10 @@
   @SerializedName("id")
   String id;
 
+  /** The line items purchased by the customer. */
+  @SerializedName("line_items")
+  List<Session.LineItem> lineItems;
```

Comment:
I think this needs to be separate from the Price changes. The problem is that Price can ship but the LineItem can't ship. Right now this is an inner class in Java but it's supposed to point to a top-level resource called `LineItem` and it will be a breaking change